tools/pot/openvino/tools/pot/configs/hardware/gna.json

```diff
@@ -168,6 +168,13 @@
                 "outputs": "q16_o"
             }
         },
+        {
+            "type": "SoftSign",
+            "quantization": {
+                "activations": "q32_a",
```

Comment:
We have a GNA3 config as well. Can you also introduce the same changes there? No additional tests are required.